Snabble/Core/Payment/PaymentMethodDetails.swift

```diff
@@ -446,50 +500,55 @@ public extension PaymentMethod {
     }
 }
 
-public struct PaymentMethodDetail: Codable, Equatable {
+public struct PaymentMethodDetail: Equatable {
+    public let id: UUID
```

Comment:
Anstatt `UUID()` in jeder Init zu schreiben, würde ich empfehlen es einmalig bei der Deklaration zu initialisieren. So kann bei keiner Init ein Fehler passieren, da sie potentiell anders initialisiert wird.

Reply:
Dann funktioniert aber `init(from:)` nicht mehr, und wir wollen womöglich dass sich andere Stellen die ID einer Paymentmethode merken können.